src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py

```diff
@@ -398,6 +410,43 @@ def ensure_container_insights_for_monitoring(
             f"/subscriptions/{cluster_subscription}/resourceGroups/{cluster_resource_group_name}/"
             f"providers/Microsoft.Insights/dataCollectionRules/{dataCollectionRuleName}"
         )
+
+        dataCollectionEndpointName = f"MSCI-{location}-{cluster_name}"
+        # Max length of the DCE name is 64 chars
+        dataCollectionEndpointName = dataCollectionEndpointName[0:64]
+        dce_resource_id = None
+
+        if azure_monitor_private_link_scope_resource_id is not None:
+            dce_resource_id = (
+            f"/subscriptions/{cluster_subscription}/resourceGroups/{cluster_resource_group_name}/"
+            f"providers/Microsoft.Insights/dataCollectionEndpoints/{dataCollectionEndpointName}"
+            )
+            dce_url = cmd.cli_ctx.cloud.endpoints.resource_manager + \
+                f"{dce_resource_id}?api-version=2022-06-01"
```

Comment:
Why use an old api version?

Reply:
> Why use an old api version?

This latest API version - https://learn.microsoft.com/en-us/rest/api/monitor/data-collection-endpoints?view=rest-monitor-2022-06-01 and we are using the same API version across all other onboardings.